codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/RulesBasedAuthSchemeResolverGenerator.kt

```diff
@@ -125,6 +128,34 @@ class RulesBasedAuthSchemeResolverGenerator {
                         )
                         write("validAuthOptions.append(sigV4Option)")
                         dedent()
```

Comment:
When S3 Express is used as the auth scheme, all the identity properties below need to be set so the S3 Express identity resolver has what it needs to get S3 Express credentials.